va/va.h

```diff
@@ -2931,6 +2960,15 @@ typedef struct _VAEncMiscParameterCustomRoundingControl {
         uint32_t    value;
     }   rounding_offset_setting;
 } VAEncMiscParameterCustomRoundingControl;
+typedef struct _VAEncDeltaQpPerBlockBuffer {
+    /*
+     * \brief This structure holds delta QP for each block in raster scan order.
+     * The size of block must be queried for the specified codec, like HEVC.
+     * Buffer size shall be sufficient to fit frame to be encoded.
+     * If qp buffer and delta qp buffer are provided at the same time, qp buffer will be ignored.
+     */
+    int16_t delta_qp;
+} VAEncDeltaQpPerBlockBuffer;
```

Comment:
I wonder whether it is needed. seems the description in VAEncDeltaQpPerBlockBufferType is enough, and also add more comments to describe the layout and the order, such as raw raster order? , and suppose it is a perframe block, will you add per tile or per slice control?

Reply:
Remove the redundant comments.  Yes, it is a per frame block. I do not add per tile or per slice control.

Reply:
how about remove the whole structure. I dont expect anybody to use the structure to allocate, access the delta qp , right?

Reply:
Yes, remove this structure. But how to indicate the size and type of delta qp value.